shared/lruwrpr/lru_wrpr.go

```diff
@@ -0,0 +1,25 @@
+package lruwrpr
+
+import (
+	"fmt"
+	lru "github.com/hashicorp/golang-lru"
+)
+
+// New creates an LRU of the given size.
+func New(size int) *lru.Cache {
```

Comment:
Please add tests for both happy and unhappy cases

Reply:
Done